src/db.rs

```diff
@@ -1246,6 +1298,7 @@ mod tests {
             #[cfg(feature = "wal_disable")]
             wal_enabled: true,
             manifest_poll_interval: Duration::from_millis(100),
+            maximum_parallel_tasks_limit: 4,
```

Comment:
this `4` is just a placeholder for now, any suggestions for a sensible default for this?